src/mpyl/cli/build.py

```diff
@@ -270,6 +279,36 @@ def by_choice(choice: Choice):
         return release.tag_name
 
 
+def get_test_releases():
+    console = Console()
+    console.log("Fetching MPyL releases..")
+    url = "https://test.pypi.org/pypi/mpyl/json"
+    data = requests.get(url, timeout=30).json()
+    versions = list(data["releases"].keys())
+    versions = list(dict.fromkeys([version[:-4] + "*" for version in versions[-100:]]))
+    versions.sort(reverse=True)
+    return versions
+
+
+def select_version(value: str) -> str:
+    versions = get_test_releases()
+
+    def question(message: str) -> str:
+        return questionary.select(
+            message=message,
+            show_selected=True,
+            choices=versions,
+        ).ask()
+
+    if value == "prompt":
+        return question("Which version do you want to install?")
+    if value not in versions:
```

Comment:
Is it even possible to pass a value that is not in the list since the prompt has a list of choices?